src/main/java/frc/robot/commands/FeederIndexCommand.java

```diff
@@ -5,53 +5,68 @@
 
 public class FeederIndexCommand extends CommandBase {
     private boolean isBallBetweenSensors; // ball state
```

Comment:
These don't need to be private instance variables, they can just be local variables in the execute function.

Reply:
Since this file is intended to be changed by https://github.com/RoboRavens/RapidReact/pull/40 you should revert your changes to this file. Essentially copy the file from the master branch and paste it into this branch. Another way is using the git command line to revert the file back to what it was at a specific commit, if you want to investigate that route (https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision)